examples/abitest/client/abitest.cc

```diff
@@ -35,237 +36,18 @@ ABSL_FLAG(std::string, manager_address, "127.0.0.1:8888",
 ABSL_FLAG(std::vector<std::string>, module, std::vector<std::string>{},
           "Files containing the compiled WebAssembly modules (as 'backend,frontend')");
 ABSL_FLAG(std::string, test_filter, "", "Filter indicating which tests to run");
```

Comment:
rename this to `test_include` for consistency? Also explain somewhere how these two flags compose.

Reply:
Done.